extensions/amp-form/0.1/amp-form.js

```diff
@@ -229,12 +237,22 @@ export class AmpForm {
     this.form_.addEventListener(
         'submit', this.handleSubmitEvent_.bind(this), true);
     this.form_.addEventListener('blur', e => {
-      onInputInteraction_(e);
+      checkUserValidityAfterInteraction_(dev().assertElement(e.target));
       this.validator_.onBlur(e);
     }, true);
+    this.form_.addEventListener('change', e => {
```

Comment:
That's right, unless you call `form.reportValidation`, which by default happens on submit.